src/inmanta/module.py

```diff
@@ -162,6 +162,14 @@ def parse(cls: Type[TInmantaModuleRequirement], spec: str) -> TInmantaModuleRequ
             raise ValueError("Invalid Inmanta module requirement: Inmanta module names use '_', not '-'.")
         return cls(Requirement.parse(spec))
 
+    def get_pip_requirement(self) -> Requirement:
```

Comment:
I just didn't find that method. Probably because I find `ModuleSourceV2` a non-intuitive location for that method. I moved it into the `InmantaModuleRequirement` class.

Reply:
That's probably better, I don't know why I put it there in the first place. Perhaps I wrote it before I added this class and didn't think to move it.